SFS_ir/interpolate_ir.m

```diff
@@ -25,6 +25,8 @@
 %                     responses.
 %     'freqdomain'  - Interpolation in the frequency domain performed separately
 %                     for magnitude and phase.
+%     'timedomain'  - Interpolation in the time domain with cross-correlation for
+%                     estimation of TOA differences
```

Comment:
For new users it might help if you write `time of arrival (TOA)` here instead of `TOA`

Reply:
Good point, I changed that.